source/_components/telegram_bot.broadcast.markdown

```diff
@@ -27,13 +27,28 @@ telegram_bot:
       - 67890
 ```
 
-Configuration variables:
-
-- **allowed_chat_ids** (*Required*): A list of user in the `user_id` Telegram format enabled to interact to webhook
-- **api_key** (*Required*): The API token of your bot.
-- **parse_mode** (*Optional*): Default parser for messages if not explicit in message data: 'html' or 'markdown'. Default is 'markdown'.
-- **proxy_url** (*Optional*): Proxy url if working behind one (`socks5://proxy_ip:proxy_port`)
-- **proxy_params** (*Optional*): Proxy configuration parameters, as dict, if working behind a proxy (`username`, `password`, etc.)
+{% configuration %}
+allowed_chat_ids:
+  description: A list of user in the `user_id` Telegram format enabled to interact to webhook
+  required: true
+  type: list
+api_key:
+  description: The API token of your bot.
+  required: true
+  type: string
+parse_mode:
+  description: Default parser for messages if not explicit in message data: 'html' or 'markdown'.
```

Comment:
It counts lines relative to the start of the variables section declaration.
There is a `:` in the `description` key value. Quote the string.